services/transaction_body.proto

```diff
@@ -93,287 +89,296 @@ import "node_delete.proto";
+         */
+        SystemUndeleteTransactionBody systemUndelete = 21;
+
+        /**
+         * Freeze the nodes
+         */
+        FreezeTransactionBody freeze = 23;
+
+        /**
+         * Creates a topic
+         */
+        ConsensusCreateTopicTransactionBody consensusCreateTopic = 24;
+
+        /**
+         * Updates a topic
+         */
+        ConsensusUpdateTopicTransactionBody consensusUpdateTopic = 25;
+
+        /**
+         * Deletes a topic
+         */
+        ConsensusDeleteTopicTransactionBody consensusDeleteTopic = 26;
+
+        /**
+         * Submits message to a topic
+         */
+        ConsensusSubmitMessageTransactionBody consensusSubmitMessage = 27;
+
+        /**
+         * UNDOCUMENTED
+         */
+        UncheckedSubmitBody uncheckedSubmit = 28;
+
+        /**
+         * Creates a token instance
+         */
+        TokenCreateTransactionBody tokenCreation = 29;
+
+        /**
+         * Freezes account not to be able to transact with a token
+         */
+        TokenFreezeAccountTransactionBody tokenFreeze = 31;
+
+        /**
+         * Unfreezes account for a token
+         */
+        TokenUnfreezeAccountTransactionBody tokenUnfreeze = 32;
+
+        /**
+         * Grants KYC to an account for a token
+         */
+        TokenGrantKycTransactionBody tokenGrantKyc = 33;
+
+        /**
+         * Revokes KYC of an account for a token
+         */
+        TokenRevokeKycTransactionBody tokenRevokeKyc = 34;
+
+        /**
+         * Deletes a token instance
+         */
+        TokenDeleteTransactionBody tokenDeletion = 35;
+
+        /**
+         * Updates a token instance
+         */
+        TokenUpdateTransactionBody tokenUpdate = 36;
+
+        /**
+         * Mints new tokens to a token's treasury account
+         */
+        TokenMintTransactionBody tokenMint = 37;
+
+        /**
+         * Burns tokens from a token's treasury account
+         */
+        TokenBurnTransactionBody tokenBurn = 38;
+
+        /**
+         * Wipes amount of tokens from an account
+         */
+        TokenWipeAccountTransactionBody tokenWipe = 39;
+
+        /**
+         * Associate tokens to an account
+         */
+        TokenAssociateTransactionBody tokenAssociate = 40;
+
+        /**
+         * Dissociate tokens from an account
+         */
+        TokenDissociateTransactionBody tokenDissociate = 41;
+
+        /**
+         * Updates a token's custom fee schedule
+         */
+        TokenFeeScheduleUpdateTransactionBody token_fee_schedule_update = 45;
+
+        /**
+         * Pauses the Token
+         */
+        TokenPauseTransactionBody token_pause = 46;
+
+        /**
+         * Unpauses the Token
+         */
+        TokenUnpauseTransactionBody token_unpause = 47;
+
+        /**
+         * Update the metadata of one or more NFT's of a specific token type.
+         */
+        TokenUpdateNftsTransactionBody token_update_nfts = 53;
```

Comment:
This reorder moves the transactions for each service to be grouped together (so token service transactions are together, crypto service transaction are together, etc...); otherwise services end up scattered about, with some transactions for the same service quite far apart.

It could be returned to number order, but that, in my opinion, makes the specification less readable for an outside contributor.

Perhaps something to discuss more widely, as number order is something more helpful when _adding_ items, but service-grouped is more helpful when _finding_ items.